modules/internal/ChapelDistribution.chpl

```diff
@@ -165,6 +168,15 @@ module ChapelDistribution {
     proc deinit() {
     }
 
+    pragma "order independent yielding loops"
+    iter _arrs: unmanaged BaseArr {
```

Comment:
I'd advocate for renaming this but it is OK for now

Reply:
I'd like to leave it as-is. It preserves the name for a bunch of call-sites  that iterate over it.